app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    return sum([player.get_rating() for player in team])
```

Comment:
The `calculate_team_total_rating` function uses a list comprehension inside the `sum` function, which creates a redundant list. According to the checklist, you should prevent creating a redundant list by using a generator expression instead. Change `[player.get_rating() for player in team]` to `(player.get_rating() for player in team)` to improve efficiency .